CI/integration_tests/calculators/test_radial_distribution_function.py

```diff
@@ -29,6 +29,7 @@
 import pytest
 from zinchub import DataHub
 
+import mdsuite
 import mdsuite as mds
```

Comment:
Should bei either one. I prefer plain `import mdsuite as mds`

Reply:
Yep, that should be changed